src/pages/login/index.tsx

```diff
@@ -39,51 +67,60 @@ function Login() {
     }
 
     const { mutate, isLoading } = useMutation((loginFormData) => {
-        return instance.post('/auth/login', loginFormData)
+        return instance.post('http://localhost:3300/auth/login', loginFormData)
```

Comment:
instance already configured with base URL = 'http://localhost:3300' => no need to add base URL here